pkg/slice/slice_test.go

```diff
@@ -0,0 +1,79 @@
+package slice_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+	"github.com/openshift-psap/special-resource-operator/pkg/slice"
+	"helm.sh/helm/v3/pkg/chart"
+)
+
+func TestSlice(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Slice Suite")
+}
+
+var _ = Describe("Slice", func() {
+	Describe("Find", func() {
```

Comment:
Context instead of second level Describe

Reply:
Removed the top-level `Describe`.